### DISCUSSION TITLE
Language translation

### DISCUSSION THREAD

**Discussion by VX-36998120** [weltenwort/frigate-synology-dsm7#63] [Q&A]
Is there any way to change these Title languages into other languages? I will translate and replace them myself
<img width="1414" alt="WX20220826-210520@2x" src="https://user-images.githubusercontent.com/61342291/186910298-6887dfc0-3f2b-4b26-95d0-6fec034d6508.png">

**Comment by weltenwort** [comment]
Hey :wave: I responded to your question #64.
